### PR TITLE
[ch9207] introduce to pipenv

### PR DESCRIPTION
**Pull request by domdinicola**

### CODE REVIEW

**Comment by reinbach** [comment]
@domdinicola looks like the `flake8` is an issue on multiple repos at the moment?!?

File: MANIFEST.in [hunk 1]
```diff
@@ -3,6 +3,8 @@ include MANIFEST.in
 include CONTRIBUTING.md
 include LICENSE
 include *.py
+include Pipfile
+include Pipfile.lock
```
Comment (saxix) [inline MANIFEST.in]: this is not required

File: Pipfile [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+mock = "*"
+factory-boy = "*"
+freezegun = "*"
+responses = "*"
+isort = "*"
+ipython = "*"
+pdbpp = "*"
+tox = "*"
+drf-api-checker = "*"
+
```
Comment (saxix) [inline Pipfile]: better to pin dev packages to the current valuers

File: tox.ini [hunk 3]
```diff
@@ -20,6 +20,7 @@ setenv =
 ;   RUNNING_UNDER_TOX=1
 
 commands =
+        pipenv install -d --system --deploy --ignore-pipfile
         ./runtests.sh
```
Comment (saxix) [inline tox.ini]: Do NOT use `--system` in tox

**Comment by domdinicola** [comment]
build was failing for incompatibility of tests with factory-boy==2.11, so for now factory-boy has been frozen to current version (2.8).
Upgrade of factory-boy will be performed in another PR

**Comment by domdinicola** [comment]
@reinbach @saxix is it good for you know?

File: Pipfile [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+PyPDF2 = "==1.26.0"
+PyYAML = "==3.12"
+unicef_attachments = "==0.4.2"
+unicef_notification = "==0.2.0"
+unicef_restlib = "==0.3.8"
+unicef_snapshot = "==0.2.1"
+Pillow = "==5.3.0"
+
+[requires]
+python_version = "3.7"
```
Comment (reinbach) [inline Pipfile]: Have we moved to python `3.7`?

File: runtests.sh [hunk 5]
```diff
@@ -20,5 +20,5 @@ fi
 # Run unittests and coverage report
 coverage erase
 time coverage run manage.py test --noinput --keepdb "$@"
-coverage report -m
+time coverage report -m
```
Comment (reinbach) [inline runtests.sh]: Not sure we care that much about timing the coverage report generation

**Comment by robertavram** [comment]
I think all these changes were included in the ci updates pr, if I'm wrong please resolve conflicts and re-open :)

**Pull request closed without merging**